src/lib/banal/Struct-Output.hpp

```diff
@@ -78,6 +78,43 @@ void printFileEnd(ostream *, FileInfo *);
 void printProc(ostream *, ostream *, string, FileInfo *, GroupInfo *,
 	       ProcInfo *, HPC::StringTable & strTab);
 
+void printBlockAndInstructionOffset(ostream * os, string file_name);
+
```

Comment:
these two functions don't belong in a header file.

moreover, they are general utilities. they don't belong in Struct-Output.cpp either. who uses them?

Reply:
I've already marked them. I'll remove them in the upcoming update.